newpotato/hitl.py

```diff
@@ -404,6 +417,66 @@ def store_triplet(
         logging.info(f"appending to triplets: {pred}, {args}")
         self.text_to_triplets[text].append((Triplet(pred, args), positive))
 
+    def store_triplets_from_annotation(self, data):
```

Comment:
I would like to suggest that we continue to add docstrings and types to our functions. For instance, when I look at some of these functions, I am not entirely certain what the "data" parameter represents. This issue is applicable to all of the functions listed below, so it would be highly beneficial to provide more clarity in our code. I suggest that we adhere to these guidelines in at least the core package.

Reply:
Thanks for the reminder, I just pushed several docstrings together with a few bugfixes

Reply:
Thanks, I now added typehints everywhere